apps/passport-server/src/services/devconnect/organizerSync.ts

```diff
@@ -762,15 +837,20 @@ export class OrganizerSync {
             );
           }
           const email = (attendee_email || order.email).toLowerCase();
+          const checkin_timestamp =
+            checkins.length > 0 ? new Date(checkins[0].datetime) : null;
```

Comment:
re: `new Date(checkins[0].datetime)` - shouldn't `checkins[0].datetime` itself be an instance of `Date` already?